test/parallel/test-tls-disable-renegotiation.js

```diff
@@ -11,8 +12,8 @@ if (!common.hasCrypto)
 const tls = require('tls');
 
 const options = {
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent1-cert.pem`)
+  key: fs.readFileSync(`${fixtures.fixturesDir}/keys/agent1-key.pem`),
```

Comment:
This could be using method `fixtures.readKey` instead of `fs.readFileSync` to get the keys.